bats/tests/containers/switch-engines.bats

```diff
@@ -0,0 +1,54 @@
+# Test case 20
+
+setup() {
+    load '../helpers/load'
+}
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'start container engine' {
+    start_container_engine
+    wait_for_container_engine
+}
+
+@test "verify we're on docker" {
+    engineName="$(rdctl list-settings | jq -r .containerEngine.name)"
+    case $engineName in
+    containerd)
+        rdctl set --container-engine.name moby
+        wait_for_container_engine
+        ;;
+    esac
+}
+
+@test "pull rancher" {
+    docker run --privileged -d --restart=no -p 8080:80 -p 8443:443 rancher/rancher
+    run docker images --format '{{json .Repository}}
+    assert_output --partial "rancher/rancher"
```

Comment:
If you only check that the image has been pulled, then you don't need to run it.
And maybe pull a smaller image, like `busybox`, which is just 2.5MiB instead of 550MiB.

```suggestion
    docker pull busybox
    run docker images --format '{{json .Repository}}
    assert_success
    assert_line '"busybox"'
```

Reply:
was draft